ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -6,12 +6,14 @@
 #include "react/renderer/components/image/ImageEventEmitter.h"
 
 #include "ReactSkia/components/RSkComponentImage.h"
+#include "ReactSkia/components/RSkComponent.h"
```

Comment:
RSkComponentImage.h already includes RSkComponent.h. so this include is not required.

Reply:
Yes not required.